server/middleware.json

```diff
@@ -0,0 +1,31 @@
+{
+  "initial:before": {
+    "loopback#favicon": {}
+  },
+  "initial": {
+    "compression": {},
```

Comment:
Is there a reason loopback-workspace itself needs any middleware? If so, they'll need to be added as dependencies.